CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## 1.10.5 - 2023-04-21
+
+### Changed
+
+- Update backup templates by @alexweininger in [#3602](https://github.com/microsoft/vscode-azurefunctions/pull/3602)
+
+### Fixed
+
+- Remove azureWebJobsStorage validation on deploy by @MicroFish91 in [#3647](https://github.com/microsoft/vscode-azurefunctions/pull/3647)
```

Comment:
Should this be in ### Fixed, ### Changed, or ### Removed? 🤔💭

Reply:
### Shrug